openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -33,12 +33,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
 
+        course_keys = []
         if options['all']:
-            # Have CourseOverview generate course overviews for all
-            # the courses in the system.
-            CourseOverview.get_all_courses(force_reseeding=True)
+            course_keys = [course.id for course in modulestore().get_courses()]
```

Comment:
Why doesn't this still use get_all_courses? If there is a reason it can't, it is not clear from the name.

(reviewing from phone, so I can't see all code)